linkchecker

```diff
@@ -419,70 +580,83 @@ if options.output:
     logtype = logtype.lower()
     if logtype not in linkcheck.logger.LoggerNames:
         print_usage(
-     _("Unknown logger type %(type)r in %(output)r for option %(option)s") % \
-      {"type": logtype, "output": options.output, "option": "'-o, --output'"})
-    if logtype != 'none' and not has_encoding(encoding):
+            _("Unknown logger type %(type)r in %(output)r for option %(option)s")
+            % {"type": logtype, "output": options.output, "option": "'-o, --output'"}
```

Comment:
This is actually a genius solution, to prevent translators from translating option names unnecessarily/misleadingly!